daemon/mgr/container.go

```diff
@@ -427,7 +430,23 @@ func (mgr *ContainerManager) List(ctx context.Context) ([]*types.ContainerInfo,
 	return cis, nil
 }
 
-// Get the detailed information of container
+// First returns the first container found in the store by a given filter.
```

Comment:
First() is not a common function, I think We don't need First(), because We can add a filter for "List()".